create-env

```diff
@@ -8,7 +8,7 @@ gfal2_bindings_version=1.12.2
 gfal2_util_version=1.8.1
 rucio_version=1.23.14
 ## REMEMBER THE 'v' IN CUTAX_VERSION!! (unless it is 'latest')
-cutax_version=v1.16.0
+cutax_version=latest
```

Comment:
This is just to test the latest cutax. We do need to release a new cutax before merge this PR, and change this line back.